### ISSUE TITLE
amazing work!, but low fps.

### ISSUE THREAD

**Issue by pushqrdx** [weolar/miniblink49#172]
the project is awesome but when I run CSS animations like [anime.js](http://animejs.com/) it renders out so slowly. I wonder if it is a limitation or something that can be fixed.

**Comment by weolar** [comment]
Yes, Now miniblink can render by GDI only,  because GPU acceleration on windows has many compatibility problems in particular windows XP.
but, there's opportunity to improve miniblink' performance , I'm doing this recently.

**Comment by pushqrdx** [comment]
@weolar great! achieving 60 fps is kinda essential to make it a viable solution, cause right now the performance is too slow. Maybe having GPU acceleration for all platforms except Windows XP?

**Comment by weolar** [comment]
Yes, this is a solution.
Before I don't implement GPU acceleration, because :
1, some GPU will cause crash. chromium resolve this by maintain a GPU blacklist.
2, GPU acceleration on layer window( or windowless software) is difficulty to implement.
so, I try to improve  in three ways: 
1, change blink thread to other thread(now it is on ui thread).
2, improve cc layer performance. (now cc layer implement by myself, not chromium)
3, GPU acceleration on some machine 

the first way and second way is high priority, and finally three way.

**Comment by pushqrdx** [comment]
@weolarI am not C/C++ savvy and I hope I could help, In fact, I will try my best doing so cause mini-blink can become the absolute best solution for many developers including C# (me :D).

**Comment by weolar** [comment]
Oh, Welcome!  I need some one to port miniblink to other Language.
You can help me, and this is some archives :

C#：
https://github.com/E024/MiniBlinkPinvoke
https://github.com/E024/MiniBlinkPinvokeDemo
https://gitee.com/kyozy/miniblinknet

delphi：
https://github.com/yangyxd/wke-for-delphi
https://gitee.com/LangjiApp/Wke4Delphi.git
